12-successful-survey-data-analysis.Rmd

```diff
@@ -141,20 +141,23 @@ anscombe_tidy %>%
   )
 ```
 
-The summary results for these four sets are nearly identical! Based on this, we might assume that each distribution is similar. Let's look at a data visualization to see if our assumption is correct.
+The summary results for these four sets are nearly identical! Based on this, we might assume that each distribution is similar. Let's look at a graphical visualization to see if our assumption is correct (see Figure \@ref(fig:recommendations-anscombe-plot)).
 
 ```{r}
 #| label: recommendations-anscombe-plot
+#| warning: false
+#| error: false
+#| fig.cap: "Plot of Anscombe's Quartet data and the importance of reveiwing data graphically"
+#| fig.alt: "This figure shows four plots one for each of Anscombe's sets. The upper left plot is a plot of set I and has a trend line with a slope of 0.5 and an intercept of 3. The data points are distributed evenly around the trend line. The upper right plot is a plot of set II and has the same trend line as set I. The data points are curved around the trend line. The lower left plot is a plot of set III and has the same trend line as set I.  The data points closely followly the trend line with one outlier where the y-value for the point is much larger than the others.  The lower right plot is a plot of set IV and has the same trend line as set I. The data points all share the same x-value but different y-values with the exception of one data point, which has a much larger value for both y and x values."
+
 ggplot(anscombe_tidy, aes(x, y)) +
```

Comment:
This output still provides a message about "`geom_smooth()` using formula = 'y~x'" Should we try to remove this?  I'm not sure what option needs to be added to the code chunk to remove that.

Reply:
Added necessary option of message: false.